tests/stress/test_stress_routes.py

```diff
@@ -19,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('t0', 'm0')
+    pytest.mark.topology('t0', 'm0', 'mx')
```

Comment:
What is diff btw test_route_perf?

Reply:
route_perf: add routes by swss_config; stress_routes: add routes by announce_routes.py